src/foraging_gui/Foraging.py

```diff
@@ -141,6 +143,33 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
             self._ReconnectBonsai()   
         logging.info('Start up complete')
     
+    def get_file_git_infor(self,file_path):
```

Comment:
This information already gets generated, but only added to the log

Reply:
https://github.com/AllenNeuralDynamics/dynamic-foraging-task/blob/main/src/foraging_gui/Foraging.py#L3361

Reply:
I combined these two functions.